clients/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
In Eaglewatch, we're currently not adding test cases because it is a task on its own to add test cases for the many existing functionalities. Let's be different here and any other future projects. Add a test case that tests our model.

Reply:
Test cases for full CRUD as a practice.